src/main/java/sh/okx/railswitch/database/SQLiteConnectionPool.java

```diff
@@ -6,15 +6,26 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 
+/**
+ * Database connection implementation for SQLite databases.
+ */
 public class SQLiteConnectionPool implements ConnectionPool {
```

Comment:
As far as I'm concerned we don't need to maintain this at all

Reply:
Agreed, in the 1.14 version we can remove all the database stuff and rely on player settings, though we should ensure that player settings wont end up causing the same problem that NameLayer did.